modules/remove-central-attached-pv-operator.adoc

```diff
@@ -3,7 +3,7 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="remove-central-attached-pv-operator_{context}"]
-= Remove Central-attached PV using the {product-title-short} Operator
+= Remove Central-attached PV using the {product-title-short} Operator (version 4.1 and later)
```

Comment:
Procedure titles should be gerund phrases per mod docs guidance.

Reply:
That was existing content, which I was generally trying to avoid restructuring. I was just asked to specify the version info.